### PR TITLE
[WIP] Implementation Questionnaire

### PR DESCRIPTION
**Pull request by gen0083**
## Issue
- #77 

## Overview (Required)
This is still on the way. We are working now.

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched2017/view/customview/QuestionnaireView.java [hunk 1]
```diff
@@ -40,6 +56,7 @@ public QuestionnaireView(Context context, AttributeSet attrs, int defStyleAttr)
         TypedArray a = context.obtainStyledAttributes(attrs, R.styleable.QuestionnaireView);
         title = a.getString(R.styleable.QuestionnaireView_questionnaireTitle);
         description = a.getString(R.styleable.QuestionnaireView_questionnaireDescription);
+        hasOtherField = a.getBoolean(R.styleable.QuestionnaireView_hasOtherField, false);
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/customview/QuestionnaireView.java]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
error: cannot find symbol

File: app/src/main/res/values/strings_questionnaire.xml [hunk 2]
```diff
@@ -11,13 +11,96 @@
...
+        <item>Designer</item>
+        <item>Marketing</item>
+        <item>HR</item>
+        <item>Media</item>
+        <item>Teacher</item>
+        <item>Student</item>
+        <item>Secret</item>
+        <item>Other</item>
+    </string-array>
+    <string name="questionnaire_job_title">Please tell me the job title?</string>
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_job_title" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 3]
```diff
@@ -11,13 +11,96 @@
...
+        <item>Other</item>
+    </string-array>
+    <string name="questionnaire_attendance">Have you attend past DroidKaigi?</string>
+    <string-array name="questionnaire_keys_attendance">
+        <item>2015</item>
+        <item>2016</item>
+        <item>First time</item>
+    </string-array>
+    <string name="questionnaire_found">How do you know DroidKaigi?</string>
+    <string-array name="questionnaire_keys_found">
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_keys_found appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 4]
```diff
@@ -11,13 +11,96 @@
...
+        <item>Secret</item>
+        <item>Other</item>
+    </string-array>
+    <string name="questionnaire_attendance">Have you attend past DroidKaigi?</string>
+    <string-array name="questionnaire_keys_attendance">
+        <item>2015</item>
+        <item>2016</item>
+        <item>First time</item>
+    </string-array>
+    <string name="questionnaire_found">How do you know DroidKaigi?</string>
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.string.questionnaire_found appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 5]
```diff
@@ -11,13 +11,96 @@
...
     </string-array>
-    <string-array name="questionnaire_values_age">
-        <item>10代</item>
-        <item>20代</item>
-        <item>30代</item>
-        <item>40代</item>
-        <item>50代</item>
-        <item>60代</item>
-        <item>答えたくない</item>
+    <string name="questionnaire_android_user">How long have you been using Android?</string>
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_android_user" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 2 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.string.questionnaire_job_title appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 6]
```diff
@@ -11,13 +11,96 @@
...
+        <item>Business owner</item>
+        <item>Manager</item>
+        <item>Lead, Senior, Chief Engineer</item>
+        <item>Engineer</item>
+        <item>Student(with programming experience)</item>
+        <item>Student(without programming experience)</item>
+        <item>Secret</item>
+        <item>Other</item>
+    </string-array>
+    <string name="questionnaire_attendance">Have you attend past DroidKaigi?</string>
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_attendance" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 7]
```diff
@@ -11,13 +11,96 @@
...
+    <string-array name="questionnaire_keys_attendance">
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_keys_attendance" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 5 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.string.questionnaire_android_user appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 8]
```diff
@@ -11,13 +11,96 @@
...
-    <string-array name="questionnaire_values_age">
-        <item>10代</item>
-        <item>20代</item>
-        <item>30代</item>
-        <item>40代</item>
-        <item>50代</item>
-        <item>60代</item>
-        <item>答えたくない</item>
+    <string name="questionnaire_android_user">How long have you been using Android?</string>
+    <string-array name="questionnaire_keys_android_user">
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_keys_android_user" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/donottranslate_questionnaire_values.xml [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+        <item>なし</item>
+        <item>〜半年未満</item>
+        <item>半年〜1年未満</item>
+        <item>1年〜2年未満</item>
+        <item>2年〜3年未満</item>
+        <item>3年〜4年未満</item>
+        <item>4年〜5年未満</item>
+        <item>5年以上</item>
+    </string-array>
+    <string-array name="questionnaire_values_android_developer">
```
Comment (konifar) [inline app/src/main/res/values/donottranslate_questionnaire_values.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_values_android_developer appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 10]
```diff
@@ -11,13 +11,96 @@
...
+        <item>Marketing</item>
+        <item>HR</item>
+        <item>Media</item>
+        <item>Teacher</item>
+        <item>Student</item>
+        <item>Secret</item>
+        <item>Other</item>
+    </string-array>
+    <string name="questionnaire_job_title">Please tell me the job title?</string>
+    <string-array name="questionnaire_keys_job_title">
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_keys_job_title appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 11]
```diff
@@ -11,13 +11,96 @@
...
+        <item>Guest</item>
+        <item>Other</item>
+    </string-array>
+    <string name="questionnaire_ticket_price">What do you think about the ticket price?</string>
+    <string-array name="questionnaire_keys_ticket_price">
+        <item>Expensive</item>
+        <item>Reasonable</item>
+        <item>Inexpensive</item>
+    </string-array>
+    <string name="questionnaire_ticket_with_after_patry">What do you think about the ticket with after party?</string>
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_ticket_with_after_patry" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 12]
```diff
@@ -11,13 +11,96 @@
...
+        <item>Mailing list</item>
+        <item>Podcast</item>
+        <item>News</item>
+        <item>Blog</item>
+        <item>group chat</item>
+        <item>co-worker/friends</item>
+        <item>Internet advertisement</item>
+        <item>Other</item>
+    </string-array>
+    <string name="questionnaire_ticket">Which ticket did you get?</string>
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_ticket" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 13]
```diff
@@ -11,13 +11,96 @@
...
+        <item>less than a half year</item>
+        <item>0.5~1 year</item>
+        <item>1~2 years</item>
+        <item>2~3 years</item>
+        <item>3~4 years</item>
+        <item>4~5 years</item>
+        <item>more than 5 years</item>
     </string-array>
+    <string name="questionnaire_android_developer">How long have you been developing Android app?</string>
+    <string-array name="questionnaire_keys_android_developer">
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_keys_android_developer" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 4 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_found" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 14]
```diff
@@ -11,13 +11,96 @@
...
+        <item>Reasonable</item>
+        <item>Inexpensive</item>
+    </string-array>
+    <string name="questionnaire_ticket_with_after_patry">What do you think about the ticket with after party?</string>
+    <string-array name="questionnaire_keys_ticket_with_after_patry">
+        <item>Good</item>
+        <item>Normal</item>
+        <item>Bad</item>
+    </string-array>
+    <string name="questionnaire_ticket_price_opinion">Do you have any opinion about the ticket price?</string>
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_ticket_price_opinion" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/donottranslate_questionnaire_values.xml [hunk 15]
```diff
@@ -0,0 +1,90 @@
...
+        <item>デザイナー</item>
+        <item>営業</item>
+        <item>人事</item>
+        <item>メディア関係</item>
+        <item>教育関係</item>
+        <item>学生</item>
+        <item>答えたくない</item>
+        <item>__other_option__</item>
+    </string-array>
+    <string-array name="questionnaire_values_job_title">
```
Comment (konifar) [inline app/src/main/res/values/donottranslate_questionnaire_values.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_values_job_title appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 14 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.string.questionnaire_ticket_price_opinion appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 6 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.string.questionnaire_attendance appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 3 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_keys_found" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 16]
```diff
@@ -11,13 +11,96 @@
...
+        <item>None</item>
+        <item>less than a half year</item>
+        <item>0.5~1 year</item>
+        <item>1~2 years</item>
+        <item>2~3 years</item>
+        <item>3~4 years</item>
+        <item>4~5 years</item>
+        <item>more than 5 years</item>
     </string-array>
+    <string name="questionnaire_android_developer">How long have you been developing Android app?</string>
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_android_developer" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/donottranslate_questionnaire_values.xml [hunk 17]
```diff
@@ -0,0 +1,90 @@
...
+        <item>学生の方：主な作業としてプログラムは書かない</item>
+        <item>答えたくない</item>
+        <item>__other_option__</item>
+    </string-array>
+    <string-array name="questionnaire_values_attendance">
+        <item>2015に参加した</item>
+        <item>2016に参加した</item>
+        <item>初めて参加した</item>
+    </string-array>
+    <string-array name="questionnaire_values_found">
```
Comment (konifar) [inline app/src/main/res/values/donottranslate_questionnaire_values.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_values_found appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 18]
```diff
@@ -11,13 +11,96 @@
...
+        <item>Regular(¥8,000)</item>
+        <item>Early Bird: incl. AfterParty (¥8,000)</item>
+        <item>Students: incl. AfterParty (¥4,000)</item>
+        <item>Regular: incl. AfterParty (¥12,000)</item>
+        <item>From presenter</item>
+        <item>Presenter</item>
+        <item>Guest</item>
+        <item>Other</item>
+    </string-array>
+    <string name="questionnaire_ticket_price">What do you think about the ticket price?</string>
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.string.questionnaire_ticket_price appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 11 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.string.questionnaire_ticket_with_after_patry appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 8 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_keys_android_user appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 12 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.string.questionnaire_ticket appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 19]
```diff
@@ -11,13 +11,96 @@
...
+        <item>None</item>
+        <item>less than a half year</item>
+        <item>0.5~1 year</item>
+        <item>1~2 years</item>
+        <item>2~3 years</item>
+        <item>3~4 years</item>
+        <item>4~5 years</item>
+        <item>more than 5 years</item>
+    </string-array>
+    <string name="questionnaire_job_category">Please tell me the job category?</string>
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_job_category" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 18 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_ticket_price" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 20]
```diff
@@ -11,13 +11,96 @@
...
+        <item>Early Bird: incl. AfterParty (¥8,000)</item>
+        <item>Students: incl. AfterParty (¥4,000)</item>
+        <item>Regular: incl. AfterParty (¥12,000)</item>
+        <item>From presenter</item>
+        <item>Presenter</item>
+        <item>Guest</item>
+        <item>Other</item>
+    </string-array>
+    <string name="questionnaire_ticket_price">What do you think about the ticket price?</string>
+    <string-array name="questionnaire_keys_ticket_price">
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_keys_ticket_price appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 21]
```diff
@@ -11,13 +11,96 @@
...
+        <item>less than a half year</item>
+        <item>0.5~1 year</item>
+        <item>1~2 years</item>
+        <item>2~3 years</item>
+        <item>3~4 years</item>
+        <item>4~5 years</item>
+        <item>more than 5 years</item>
+    </string-array>
+    <string name="questionnaire_job_category">Please tell me the job category?</string>
+    <string-array name="questionnaire_keys_job_category">
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_keys_job_category" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 16 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.string.questionnaire_android_developer appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 7 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_keys_attendance appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 10 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_keys_job_title" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/donottranslate_questionnaire_values.xml [hunk 22]
```diff
@@ -0,0 +1,90 @@
...
+        <item>なし</item>
+        <item>〜半年未満</item>
+        <item>半年〜1年未満</item>
+        <item>1年〜2年未満</item>
+        <item>2年〜3年未満</item>
+        <item>3年〜4年未満</item>
+        <item>4年〜5年未満</item>
+        <item>5年以上</item>
+    </string-array>
+    <string-array name="questionnaire_values_job_category">
```
Comment (konifar) [inline app/src/main/res/values/donottranslate_questionnaire_values.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_values_job_category appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 20 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_keys_ticket_price" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml [hunk 23]
```diff
@@ -11,13 +11,96 @@
...
+        <item>Podcast</item>
+        <item>News</item>
+        <item>Blog</item>
+        <item>group chat</item>
+        <item>co-worker/friends</item>
+        <item>Internet advertisement</item>
+        <item>Other</item>
+    </string-array>
+    <string name="questionnaire_ticket">Which ticket did you get?</string>
+    <string-array name="questionnaire_keys_ticket">
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_keys_ticket appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 19 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.string.questionnaire_job_category appears to be unused [UnusedResources]

**Comment by konifar** [comment]
Thanks! Please tell me when I can review 👍

File: app/src/main/res/values/strings_questionnaire.xml [hunk 24]
```diff
@@ -11,13 +11,96 @@
...
+        <item>Other</item>
+    </string-array>
+    <string name="questionnaire_ticket_price">What do you think about the ticket price?</string>
+    <string-array name="questionnaire_keys_ticket_price">
+        <item>Expensive</item>
+        <item>Reasonable</item>
+        <item>Inexpensive</item>
+    </string-array>
+    <string name="questionnaire_ticket_with_after_patry">What do you think about the ticket with after party?</string>
+    <string-array name="questionnaire_keys_ticket_with_after_patry">
```
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_keys_ticket_with_after_patry" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/donottranslate_questionnaire_values.xml [hunk 25]
```diff
@@ -0,0 +1,90 @@
...
+        <item>Regular(¥8,000)</item>
+        <item>Early Bird: incl. AfterParty (¥8,000)</item>
+        <item>Students: incl. AfterParty (¥4,000)</item>
+        <item>Regular: incl. AfterParty (¥12,000)</item>
+        <item>From presenter</item>
+        <item>Presenter</item>
+        <item>Guest</item>
+        <item>Other</item>
+    </string-array>
+    <string-array name="questionnaire_value_ticket_with_after_patry">
```
Comment (konifar) [inline app/src/main/res/values/donottranslate_questionnaire_values.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_value_ticket_with_after_patry appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 21 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_keys_job_category" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 24 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_keys_ticket_with_after_patry appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 7 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_keys_attendance appears to be unused [UnusedResources]

File: app/src/main/res/values/donottranslate_questionnaire_values.xml (hunk 22 above)
Comment (konifar) [inline app/src/main/res/values/donottranslate_questionnaire_values.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_values_job_category appears to be unused [UnusedResources]

File: app/src/main/res/values/donottranslate_questionnaire_values.xml [hunk 26]
```diff
@@ -0,0 +1,90 @@
...
+        <item tools:ignore="TypographyEllipsis">チケットサービスのメール（connpass, doorkeeper, etc...）</item>
+        <item tools:ignore="TypographyEllipsis">ポッドキャスト(dex.fm, デブ☆ラジ, etc...)</item>
+        <item>ニュースサイト</item>
+        <item>ブログ・日記</item>
+        <item>社内・グループチャット</item>
+        <item>同僚・知人からの直接紹介</item>
+        <item>インターネット広告・AdWords</item>
+        <item>__other_option__</item>
+    </string-array>
+    <string-array name="questionnaire_values_ticket">
```
Comment (konifar) [inline app/src/main/res/values/donottranslate_questionnaire_values.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_values_ticket appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 23 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_keys_ticket appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 10 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_keys_job_title" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 16 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.string.questionnaire_android_developer appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 23 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_keys_ticket" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 21 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_keys_job_category appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 13 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_keys_android_developer appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 20 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_keys_ticket_price" is not translated in "ja" (Japanese) [MissingTranslation]

File: app/src/main/res/values/donottranslate_questionnaire_values.xml [hunk 27]
```diff
@@ -0,0 +1,90 @@
...
+        <item>経営者</item>
+        <item>マネージャ</item>
+        <item>リード、シニア、チーフエンジニア</item>
+        <item>エンジニア</item>
+        <item>学生の方：主な作業としてプログラムを書く</item>
+        <item>学生の方：主な作業としてプログラムは書かない</item>
+        <item>答えたくない</item>
+        <item>__other_option__</item>
+    </string-array>
+    <string-array name="questionnaire_values_attendance">
```
Comment (konifar) [inline app/src/main/res/values/donottranslate_questionnaire_values.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_values_attendance appears to be unused [UnusedResources]

File: app/src/main/res/values/donottranslate_questionnaire_values.xml [hunk 28]
```diff
@@ -0,0 +1,90 @@
...
+    <string-array name="questionnaire_values_age">
+        <item>10代</item>
+        <item>20代</item>
+        <item>30代</item>
+        <item>40代</item>
+        <item>50代</item>
+        <item>60代</item>
+        <item>答えたくない</item>
+    </string-array>
+    <string-array name="questionnaire_values_android_user">
```
Comment (konifar) [inline app/src/main/res/values/donottranslate_questionnaire_values.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_values_android_user appears to be unused [UnusedResources]

File: app/src/main/res/values/strings_questionnaire.xml (hunk 19 above)
Comment (konifar) [inline app/src/main/res/values/strings_questionnaire.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.string.questionnaire_job_category appears to be unused [UnusedResources]

File: app/src/main/res/values/strings.xml [hunk 29]
```diff
@@ -103,12 +103,120 @@
...
+        <item>20~29</item>
+        <item>30~39</item>
+        <item>40~49</item>
+        <item>50~59</item>
+        <item>60~</item>
+        <item>secret</item>
+    </string-array>
+
+    <string name="questionnaire_title_android_user">How long have you been using Android?</string>
+    <string-array name="questionnaire_labels_android_user">
```
Comment (konifar) [inline app/src/main/res/values/strings.xml]: **[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_labels_android_user appears to be unused [UnusedResources]

**Comment by gen0083** [comment]
Now implement views for questionnaire page.
I implement views type using on google form:

- CheckBox
- RadioButton
- RadioButton with other field
- TextInput
- Rating(using FeedbackRankingView thanks!)

https://docs.google.com/forms/d/e/1FAIpQLSfAD2H_ePyLV3wqCEXwlxS9dGu3OKtr-IrxbGoLo4QYzJh_dw/viewform

You can check these views on QuestionnaireFragment.
Selected value was bound to Questionnaire model in QuestionnaireViewModel.
You can see what values are set when "Send Questionnaire" button clicked.
(logged only to logcat)

Now I don't add all of questionnaire item because I have not implemented strings.xml yet.
If you can review please advice me about view's implementation.
Thanks.

---

日本語で補足

Webのアンケートフォームで使われている部品をViewとして作りました。
とりあえず作ったViewのすべてのタイプをQuestionnaireFragmentで試すことができます。
RadioButton with other fieldだけ挙動がまだおかしいです。
（その他のEditTextをタップするとクラッシュする）

Viewの実装方法、あとは変数名の付け方あたりを中心にレビューいただけると助かります。

**Comment by konifar** [comment]
Thanks! Let me merge first 🙇

**Comment by gen0083** [comment]
Don't worry, please point out only the inevitable part if you find it 😄

**Pull request merged**